lib/view_model/active_record/update_operation.rb

```diff
@@ -158,6 +163,15 @@ def run!(deserialize_context:)
         end
       end
 
+      if self.released_children.present?
+        debug "-> #{debug_name}: Checking released children permissions"
+        self.released_children.reject(&:claimed?).each do |released_child|
```

Comment:
But can't a different subtree claim these children, before the entire process finishes?